views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -139,6 +141,20 @@ define([
             interaction.attr('patternMask', attrValue);
         };
 
+        callbacks.expectedLength = function(interaction, attrValue){
+            var newValue = parseInt(attrValue,10);
+            if(! isNaN(newValue)){
+                interaction.attr('expectedLength', attrValue);
+            }
+        };
```

Comment:
same issue as for expectedLines : the change on the interaction rendering is not immediately visible. Since the item creator adopted the wysiwyg philosophy, we need an immediate visual feedback.